IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerTokenRequest.m

```diff
@@ -63,9 +64,14 @@ - (NSDictionary *)protocolPayloadContentsWithError:(__unused NSError **)error
 
 - (NSDictionary *)protocolResumeDictionaryContents
 {
-    return @{@"scope": self.requestParameters.target ?: @"",
-             @"oidc_scope": self.requestParameters.oidcScope ?: @"",
-             MSID_SDK_NAME_KEY: MSID_MSAL_SDK_NAME};
+    NSMutableDictionary *protocolResumeDictionary = [NSMutableDictionary new];
+    [protocolResumeDictionary msidSetNonEmptyString:self.requestParameters.target ?: @"" forKey:@"scope"];
+    [protocolResumeDictionary msidSetNonEmptyString:self.requestParameters.oidcScope ?: @"" forKey:@"oidc_scope"];
+    [protocolResumeDictionary msidSetNonEmptyString:MSID_MSAL_SDK_NAME forKey:MSID_SDK_NAME_KEY];
+    [protocolResumeDictionary msidSetNonEmptyString:self.requestParameters.providedAuthority.url.absoluteString ?: self.requestParameters.authority.url.absoluteString forKey:@"provided_authority_url"];
```

Comment:
nit: do we want to just set nil for shorter code?

Reply:
addressed :)